### PR TITLE
Modulesync 2.9.0 / Make `disable_unhide` `Optional`, convert params to boolean and small fixes

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
Getting smaller commits out of this with working tests is tricky because many of them depend on each other. Let me know what your idea is.

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+RSpec.configure do |c|
+  # Readable test descriptions
+  c.formatter = :documentation
+
+  c.before :suite do
+    install_module
+    install_module_dependencies
+    hosts.each do |host|
+      if fact('os.family') == 'RedHat'
+        # Soft dep on epel for Passenger
```
Comment (alexjfisher) [inline spec/spec_helper_acceptance.rb]: Passenger?

File: manifests/packages.pp [hunk 2]
```diff
@@ -1,6 +1,16 @@
+#
+# @summary install the rkhunter package and initialize it
+#
+# @api private
+#
+# @param package_name name of the rkhunter packag that will be installed
```
Comment (alexjfisher) [inline manifests/packages.pp]: typo `packag`

**Comment by alexjfisher** [comment]
Other than a couple of nitpicks/typos, this looks *great*!  Since it contains so much, maybe we should just update the PR title to 'Migrate to Voxpupuli' and add a commit to update the metadata?  Then open a release-summary issue to explain the migration.

File: spec/spec_helper_acceptance.rb (hunk 1 above)
Comment (bastelfreak) [reply spec/spec_helper_acceptance.rb]: fixed

File: manifests/packages.pp (hunk 2 above)
Comment (bastelfreak) [reply manifests/packages.pp]: fixed

**Pull request merged**